runtime/src/app/app.ts

```diff
@@ -54,7 +54,7 @@ stores.session.subscribe(async value => {
 	if (redirect) {
 		await goto(redirect.location, { replaceState: true });
 	} else {
-		await render(branch, props, dest.page);
+		await render(branch, props, Object.assign({error: props.error}, dest.page));
```

Comment:
I wasn't sure if it was ok to use modern JS here; would it be ok to switch to the spread operator instead?

Reply:
I think it should be because this is just for modern browsers and legacy browsers should go through babel. Surprisingly I'm not seeing it done anywhere in the codebase except for server-side code